### PR TITLE
Added fluid simulation gif to the gallery

### PR DESCRIPTION
**Pull request by Wumpf**
My humble contribution to the gallery
From https://github.com/Wumpf/blub/

@lachlansneff tried mp4 (_so_ much nicer for less size) but apparently only embedded youtube videos work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ wgpu-rs can target both the natively supported backends and WASM directly.
 
 ![Cube](etc/example-cube.png) ![Shadow](etc/example-shadow.png) ![MipMap](etc/example-mipmap.png) ![Skybox](etc/example-skybox.gif)
 ![vange-rs](etc/vange-rs.png) ![Brawl](etc/brawl-attack.gif) ![GLX map](etc/glx-map.png) ![Harmony](etc/harmony-rs.jpg)
+![Blub](etc/blub-fluid-small.gif)
```
Comment (OptimisticPeach) [inline README.md:19]: To keep it tidy on the front page for desktop, why not place this between the vange-rs example and the brawl example?

**Comment by lachlansneff** [comment]
Oh, oops, my bad. Used mp4 on the wgpu gallery site, but github doesn't support them unfortunately.

File: README.md (hunk 1 above)
Comment (Wumpf) [reply README.md:19]: tried, actually, didn't look much better though. Also github already has a new design on the way (you can opt into it) which moves everything around again _sigh_

Comment (Wumpf) [reply README.md]: ah actually. I figure it does look a bit better. updated.

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ wgpu-rs can target both the natively supported backends and WASM directly.
 ## Gallery
 
 ![Cube](etc/example-cube.png) ![Shadow](etc/example-shadow.png) ![MipMap](etc/example-mipmap.png) ![Skybox](etc/example-skybox.gif)
-![vange-rs](etc/vange-rs.png) ![Brawl](etc/brawl-attack.gif) ![GLX map](etc/glx-map.png) ![Harmony](etc/harmony-rs.jpg)
+![vange-rs](etc/vange-rs.png) ![Blub](etc/blub-fluid-small.gif) ![Brawl](etc/brawl-attack.gif) ![GLX map](etc/glx-map.png) ![Harmony](etc/harmony-rs.jpg)
```
Comment (kvark) [inline README.md:18]: I'm concerned about this being in each and every checkout of the code.
Let's instead try to cut this gallery a bit (only non-animated and small images), and expand the one on `wgpu.rs`.
How do you feel about targeting the "gh-pages" branch for this?

**Pull request closed without merging**